src/lib/pickles/step_branch_data.ml

```diff
@@ -157,10 +157,33 @@ let create
     in
     let etyp =
       Impls.Step.input ~proofs_verified:max_proofs_verified
-        ~wrap_rounds:Backend.Tock.Rounds.n ~feature_flags
+        ~wrap_rounds:Backend.Tock.Rounds.n
       (* TODO *)
     in
-    Fix_domains.domains
+    let lookup_table_length_log2 =
+      let { Plonk_types.Features.range_check0
+          ; range_check1
+          ; foreign_field_add = _
+          ; foreign_field_mul
+          ; xor
+          ; rot
+          ; lookup
+          ; runtime_tables
+          } =
+        actual_feature_flags
+      in
+      let total_length =
+        ( if range_check0 || range_check1 || foreign_field_mul || rot then
```

Comment:
Can you explain the different values? I don't understand why it is 3 when xor is used and 2^8 when not used, where it is 2^12 and 0 for the other.